diff --git a/node_modules/@ohif/core/src/services/HangingProtocolService/HangingProtocolService.ts b/node_modules/@ohif/core/src/services/HangingProtocolService/HangingProtocolService.ts
index 1234567..abcdefg 100644
--- a/node_modules/@ohif/core/src/services/HangingProtocolService/HangingProtocolService.ts
+++ b/node_modules/@ohif/core/src/services/HangingProtocolService/HangingProtocolService.ts
@@ -1530,7 +1530,7 @@
     const matchingScores = [];
     let highestSeriesMatchingScore = 0;
 
-    console.log('ProtocolEngine::matchImages', studyMatchingRules, seriesMatchingRules);
+    // console.log('ProtocolEngine::matchImages', studyMatchingRules, seriesMatchingRules);
     const matchActiveOnly = this.protocol.numberOfPriorsReferenced === -1;
     this.studies.forEach((study, studyInstanceUIDsIndex) => {
       // Skip non-active if active only
@@ -1630,7 +1630,7 @@
 
     const bestMatch = matchingScores[0];
 
-    console.log('ProtocolEngine::matchImages bestMatch', bestMatch, matchingScores);
+    // console.log('ProtocolEngine::matchImages bestMatch', bestMatch, matchingScores);
 
     return {
       bestMatch,
